packages/app-project/src/screens/ProjectHomePage/components/Hero/components/WorkflowSelector/components/WorkflowSelectButton/WorkflowSelectButton.js

```diff
@@ -18,10 +27,24 @@ function WorkflowSelectButton (props) {
 
   const as = addQueryParams(url, router)
   const href = '/projects/[owner]/[project]/classify'
+  const completeness = parseInt(workflow.completeness * 100)
```

Comment:
this needs a radix of 10:

`const completeness = parseInt(workflow.completeness * 100, 10)`